ls1intum/Artemis
# `Tutorial groups`: Improve checklist consistency

### Checklist
#### General
#### Client

### Motivation and Context
When configuring tutorial groups for the first time, a checklist is used to guide the instructor through the configuration. This checklist used disabled checkboxes to indicate the current status, which is confusing and inconsistent with other checklists.

### Description
This PR reuses the exam checklist check component and makes it generally available.

### Steps for Testing
- 1 Instructor
- 1 Course

1. Open the tutorial group configuration checklist. If the course has no tutor groups yet, you can simply open it by pressing the tutorial groups button at the course management card. When groups are already configured you can reach it using the direct URL `course-management/<courseId>/tutorial-groups-checklist`
2. Verify that it works correctly and looks good.

### Review Progress
#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots
before:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/0f052689-fdb1-4548-8cfc-712e125334b7)


now:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/aadaaf61-f8cf-4ff0-a8e2-dad06623806d)
